setup.py

```diff
@@ -23,7 +23,7 @@ def read(*rnames):
 
 setup(
     name='djangosaml2_tenant',
-    version='0.13.0',
+    version='0.1.0',
```

Comment:
@alexenko : What do you think about keeping the version string to 0.2.0, so its larger than the one we forked from?